drivers/scheduler/scheduler.go

```diff
@@ -184,11 +186,17 @@ type Driver interface {
 	// AddLabelOnNode adds key value labels on the node
 	AddLabelOnNode(node.Node, string, string) error
 
-	//IsAutopilotEnabledForVolume checks if autopilot enabled for a given volume
+	// IsAutopilotEnabledForVolume checks if autopilot enabled for a given volume
 	IsAutopilotEnabledForVolume(*volume.Volume) bool
 
-	// GetSpecAppEnvVar gets app environment variable value by given key name
-	GetSpecAppEnvVar(ctx *Context, key string) string
+	// GetEvents dumps events from event storage
+	GetEvents() map[string][]Event
+
+	// ValidateAutopilotPVCEvents validates events for PVCs injected by autopilot
+	ValidateAutopilotPVCEvents(ctx *Context, coolDownPeriod int) error
```

Comment:
remove the word "PVC" from the interface so that we can use it for validate events for other scheduler objects in future. for e.g pods.